server-cli/src/main/java/org/apache/brooklyn/cli/Main.java

```diff
@@ -827,6 +827,82 @@ public ToStringHelper string() {
         }
     }
 
+    @Command(name = "clean-orphaned-locations", description = "Removes existing orphaned locations")
```

Comment:
How about name "clean-orphaned-state" and description "Removes existing orphaned persisted state (e.g. locations, policies, enrichers and feeds)".